[CS2113T-W12-1] PlaNUS

QX-CHEN opened the pull request:
PlaNUS is a desktop app for NUS students to manage their tasks and schedule, optimized for use via Command Line Interface (CLI).

QX-CHEN commented:
PlaNUS is a desktop app for NUS students to manage their tasks and schedule, optimized for use via Command Line Interface (CLI).

QX-CHEN committed:
Merge pull request #53 from QX-CHEN/JUnitTests

JUnit tests and User Guide

QX-CHEN committed:
Fix image issue and move introduction to readme

QX-CHEN committed:
Merge pull request #54 from QX-CHEN/JUnitTests

Fix image issue and move introduction to readme

arindamshivatrip committed:
Update AboutUs.md

arindamshivatrip committed:
Update AboutUs.md

QX-CHEN committed:
UG/DG

QX-CHEN committed:
Add manual testing section

QX-CHEN committed:
Merge pull request #55 from QX-CHEN/JUnitTests

Developer guide

QX-CHEN committed:
Use cases formatting

QX-CHEN committed:
Merge pull request #56 from QX-CHEN/JUnitTests

Use cases formatting

czlin98 committed:
Merge pull request #2 from AY2021S1-CS2113T-W12-1/master

Merge back

czlin98 committed:
Improve readability of messages

czlin98 committed:
Merge pull request #57 from czlin98/fix-messages

Improve readability of messages strings

QX-CHEN committed:
UG fixes

QX-CHEN committed:
Merge pull request #58 from QX-CHEN/master

UG fixes

QX-CHEN committed:
Add some newlines in UG

QX-CHEN committed:
Merge pull request #59 from QX-CHEN/master

Add some newlines in UG

mhchan163 committed:
Merge branch 'master' of https://github.com/AY2021S1-CS2113T-W12-1/tp.git

czlin98 committed:
Merge pull request #3 from AY2021S1-CS2113T-W12-1/master

Merge back

czlin98 committed:
Fix user guide

czlin98 committed:
Fix user guide again

czlin98 committed:
Merge pull request #60 from czlin98/master

Fix user guide

QX-CHEN committed:
Minor changes

QX-CHEN committed:
Merge pull request #61 from QX-CHEN/master

Minor changes

timcheahyi committed:
some tests for add and delete

timcheahyi committed:
formatting

timcheahyi committed:
formatting

QX-CHEN committed:
Merge pull request #62 from e0406981/master

some add and delete tests

timcheahyi committed:
more coverage for add and delete

timcheahyi committed:
more coverage for add and delete

timcheahyi committed:
Merge pull request #67 from e0406981/master

more coverage for add and delete

QX-CHEN committed:
Add undo function

QX-CHEN committed:
Merge branch 'master' of https://github.com/AY2021S1-CS2113T-W12-1/tp into undo

QX-CHEN committed:
Update test

QX-CHEN committed:
Merge branch 'undo'

mhchan163 committed:
Merge branch 'master' of https://github.com/AY2021S1-CS2113T-W12-1/tp.git

mhchan163 committed:
Changed Parser class to extract parameters before calling the command

mhchan163 committed:
Merge branch 'ParserRestructure'

mhchan163 committed:
Merge pull request #68 from QX-CHEN/undo

Add undo function

mhchan163 committed:
Deleted some unneccesary imports

mhchan163 committed:
Merge branch 'master' of https://github.com/AY2021S1-CS2113T-W12-1/tp.git

# Conflicts:
#	src/main/java/seedu/commands/Add.java
#	src/main/java/seedu/parser/Parser.java

mhchan163 committed:
Edited DeleteTest and AddTest accordingly:
Changed Add/Delete Constructor to parser.processRaw()

mhchan163 committed:
Corrected AddTest and DeleteTest contructor_noRawInput_throws_InvalidCommandException

mhchan163 committed:
Edited PR errors

mhchan163 committed:
More PR test errors fixed

mhchan163 committed:
Extracted parameters from List command

mhchan163 committed:
Fixed PR errors

QX-CHEN committed:
Merge branch 'master' of https://github.com/AY2021S1-CS2113T-W12-1/tp

QX-CHEN committed:
Merge pull request #69 from mhchan163/master

Extracted parameters from Command to Parser

QX-CHEN committed:
Merge branch 'master' of https://github.com/AY2021S1-CS2113T-W12-1/tp

QX-CHEN committed:
Fix monthly view last week display

QX-CHEN committed:
Add quit,previous and next in partial display

QX-CHEN committed:
Minor change

QX-CHEN committed:
Merge branch 'display'

mhchan163 committed:
Merge pull request #70 from QX-CHEN/display

Display

mhchan163 committed:
Added list to message for ADD Delete and Edit

mhchan163 committed:
Added Reminder function

mhchan163 committed:
Merge branch 'ReminderFunction'

mhchan163 committed:
Fix PR errors

mhchan163 committed:
Added Reminder command into features section

mhchan163 committed:
minor changes

mhchan163 committed:
Added images for add command

mhchan163 committed:
Added images for edit commands in features section

mhchan163 committed:
Added images to expected outcome part of the delete feature.
And rephrased list explanation

mhchan163 committed:
Minor change

mhchan163 committed:
Changed structure of Add, Edit, Delete task info

mhchan163 committed:
Added displaySingleTask method in Ui

mhchan163 committed:
small change

domaxi commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,383 @@
+and these classes are stored in the seedu.commons package.
 
 
 ## Product scope
```
Wouldn't it better if you guys can move the product scope to the front of the document to introduce the reader to the app first, rather than starting off with the complex diagrams?

domaxi commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,383 @@
+### Architecture
+
+
+![Architecture](diagrams/Architecture.png)
```
I would like to suggest to move the description to above the image, just so that the user is introduced to the image before reading them

adinata15 replied:
![image](https://user-images.githubusercontent.com/48993548/97530529-4f1a4f80-19ed-11eb-966d-1047f80da59a.png)
I suggest using a double arrow on some of the diagrams as the interaction is both ways. Aside from that, would it be possible that the `Commons` is actually linked to some other part?

domaxi commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,383 @@
+
+Commons provide utility functions and messages to be used by other components.
+
+The sequence diagram below shows a typical workflow the program.
```
Perhaps you can show which of the diagrams the text are explaining about, to give a bit more context to the user.

jusufnathanael commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,383 @@
+
+
+![UiComponent](diagrams/UiClass.png)
+
```
I think it's better to write {abstract} for abstract class rather than just put (A) in front of the name.

![image](https://user-images.githubusercontent.com/57692185/97386648-3e93a780-190f-11eb-9cec-32b6a1826cc6.png)

domaxi commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,383 @@
+### Ui component
+
+
+![UiComponent](diagrams/UiClass.png)
```
I think you can change the C, E, and A logo on the class names to the format of <<abstract>>, <<enum>>.

0xZ3RO replied:
Could have elaborated more on how the week and month structure is related to the UI. It isn't clear how they are related without checking the User Guide. Maybe you could add an image?
![](https://ay2021s1-cs2113t-w12-1.github.io/tp/diagrams/UiClass.png)

lunzard replied:
I think you can add a legend chart at the side to show what is `A`,`C`, `E` for clarification, though I understand they stand for `Abstract`, `Class `and `Enum`.
Also, adding labels to the associations could make the diagram more easy for reader to understand!

domaxi commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,383 @@
+
+
+![UiComponent](diagrams/UiClass.png)
+
```
Another thing to consider, I am not sure what the hash symbol means on some of your attributes and or methods

lunzard replied:
Hi domaxi the hash ` #` means protected in visibility.

jusufnathanael commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,383 @@
+
+
+![ComponentsSeq](diagrams/ComponentsSeq.png)
+
```
Is there supposed to be any connection between storage and taskmap?

![image](https://user-images.githubusercontent.com/57692185/97386924-e5784380-190f-11eb-8bd7-234251e39b2d.png)

jusufnathanael replied:
From Prof: Do you think it will be better to call loadTasks() from the TaskMap rather than from the main function?

adinata15 replied:
If there is no relation between the Storage+Main and the rest of the diagram, they can be separated into a different diagram.

jusufnathanael commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,383 @@
+
+
+![LogicComponent](diagrams/LogicClass.png)
+
```
For the different commands, I think it's better to name the classes as ListCommand, SearchCommand, HelpCommand, etc. rather than just the keyword themselves.

![image](https://user-images.githubusercontent.com/57692185/97387337-bd3d1480-1910-11eb-979f-1ee5d497df9d.png)

domaxi commented on docs/UserGuide.md:
```diff
@@ -1,42 +1,259 @@
+
+### <span style="color:red">Adding a task</span> : `add`
+
+Add a task to the task list and display a message.
```
Perhaps, you can add sequence diagrams to the methods and features that you mentioned. It would certainly add more value to each of the features.

jusufnathanael commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,383 @@
+
+
+![ModelComponent](diagrams/TaskMapClass.png)
+
```
For the startTime/endTime, maybe you want to move it closer to the either one of the classes rather than put it right in the middle?

![image](https://user-images.githubusercontent.com/57692185/97387675-73086300-1911-11eb-81bc-ea901dfa92fb.png)

jusufnathanael replied:
From Prof: HASH_VALUE_DIGITS is a private static method, is it necessary to add a getter method?

0xZ3RO commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,383 @@
+
+Now the Ui component is ready to take inputs from the User, and the input is processed and executed
+in the Logic component, the execution result is passed back to Ui component and displayed to the User.
+Just before the main program exits, Storage component saves the content in the TaskMap to the data file.
```
Even though it is mentioned that Storage saves the content in the TaskMap to the data file, in the architecture diagram, there isn't a link between the two.

0xZ3RO commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,383 @@
+The sequence diagram below shows a typical workflow the program.
+
+
+![ComponentsSeq](diagrams/ComponentsSeq.png)
```
Would a typical workflow start with "delete 2173"?

domaxi replied:
I am quite confused in the diagram because I cannot see when the classes are constructed. Maybe you can revise the diagram to show when the class construction also.

0xZ3RO replied:
The user input "bye" and the writeTasksToFile() functions are not connected in the diagram?

0xZ3RO replied:
Does the program destroy the objects after calling "bye"?

yeyutong811 replied:
Maybe you can include the sequence diagram of how TaskMap objects are constructed via Add Task?

domaxi commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,383 @@
+The Architecture Diagram above gives an overview of how different components interacts 
+with each other in the app. 
+
+As the app starts, the main program initialises the Ui, Logic and Storage components.
```
Perhaps you want to illustrate the startup sequence using a sequence diagram, rather than using box and arrow diagrams.

domaxi commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,383 @@
+
+
+![Architecture](diagrams/Architecture.png)
+
```
A small suggestion I would make is to include numbering for all your diagram and images, to that the user can easily reference the images and diagrams that you created.

domaxi reviewed (commented):
Overall, the developer guide is really good. It includes all the important details, and the guide is really comprehensive. Good Job Team W12 - 1!! Keep up the good stuff!

QX-CHEN committed:
Merge pull request #71 from mhchan163/master

Added Reminder command and Task info for add, edit and delete commands

czlin98 committed:
Update UG

czlin98 committed:
Merge pull request #73 from czlin98/update-ug

Update UG

lunzard commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,383 @@
+- [Non-functional requirements](#non-functional-requirements) 
+- [Glossary](#glossary)
+- [Instruction for manual testing](#instructions-for-manual-testing)
+
```
I think adding a brief introduction about PlaNUS before the Design & Implementation Section will help reader better understanding the function of each component.

QX-CHEN committed:
Minor change

QX-CHEN committed:
Merge branch 'master' of https://github.com/AY2021S1-CS2113T-W12-1/tp

QX-CHEN committed:
Fix internal links

lunzard commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,383 @@
+
+Common classes provide utility functions and string messages used by multiple components,
+and these classes are stored in the seedu.commons package.
 
 
```
It would be better if you can add more sequence diagrams to your features or user scenarios so that reader can better understand how you implemented each feature!

QX-CHEN committed:
Change purple to green for help messages

lunzard commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,383 @@
+At the initiating and closing stage of the program, the Storage component
+will read and write to the data file respectively. As shown in the diagram
+above, the data file is named as "data.json" and placed under a folder called
+"data". The Storage component also uses the Google Gson library to convert Task
```
Instead of giving the sole folder name `data`, you may paste its absolute directory here so that reader can easily find it for future development.

lunzard commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,383 @@
+The sequence diagram below shows a typical workflow the program.
+
+
+![ComponentsSeq](diagrams/ComponentsSeq.png)
```
Like what other reviewers said, the construction of objects never appeared in the sequence diagram. Maybe you can add another sequence diagram including how an object of Class `taskMap` is constructed at the start?

lunzard replied:
Also including how each object terminates in the diagrams will help the reader to understand the entire process better!

lunzard reviewed (commented):
Overall, the explanation is fine to me. It would be better if more sequence diagrams are added for stronger clarification!

QX-CHEN committed:
Change welcome message to v2.0

adinata15 commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,383 @@
+    Expected: homework task should be added to the list with default date being set 
+    to the current date.
+
+2. Add a task into the lists using add command with description and date.
+
+    - Test case: add homework d/25-12-2020  
+    Expected: homework task should be added to the list with the date set as 25-12-2020.
+
+    - Test case: add homework d/25-13-2020  
+    Expected: Since there is no 13th month, the task is not added to the list, 
+    and an error message is shown.
+
```
It might be helpful to give a brief description of what the identifier (eg: `/d`) means. For example, can give an explanation that `/d` represents the date.

adinata15 commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,383 @@
+![LogicComponent](diagrams/LogicClass.png)
+
+
+From the Ui component, the raw input from the User is passed to the Parser class.
+Parser class will process and validate the input, then it generates a specific
+command accordingly.  Specific commands extends from the abstract Command class.
+Command operates on the TaskMap and execution of Command generates a 
+CommandResult. This CommandResult is passed back to the Ui component and 
+respective content will be displayed to User based on the CommandResult. 
```
It will be helpful if the explanation is given in steps format (eg: 1,2,3,...) to improve readability.

adinata15 commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,383 @@
+|v2.0|Busy student|View tasks in a weekly and monthly view|I know what i have due for that time period|
+
+
+## Use cases
+
+### Use case: Add task
+
+Main success scenario
+
+1. User adds a task
+2. PLANus adds task
+3. PLANus shows the task added message
+
+Use case ends. 
+
+Extensions  
+   - 1a. User adds task without any date  
+Date is set to the current date  
+   - 1b. User adds tasks without any priority  
+Priority is set to low  
+   - 1c. User adds task without start and end time  
+Time is set to empty  
+   - 1d. User inputs wrong details format  
+PLANus shows invalid command message  
+
```
A sequence or object diagram can be drawn to better explain how each task/command operates.

adinata15 reviewed (commented):
Overall great job!

czlin98 committed:
Add java link to quick start

czlin98 committed:
Add java link to quick start and update notes

czlin98 committed:
Merge remote-tracking branch 'origin/update-ug' into update-ug

czlin98 committed:
Merge pull request #74 from czlin98/update-ug

Small update ug

QX-CHEN committed:
Update usage of q,w,e in partial display

QX-CHEN committed:
Merge pull request #75 from QX-CHEN/master

Update display for UG

yeyutong811 commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,383 @@
+
+1. Add a task into the list by using add command using only description
+
+    - Test case: add homework  
```
Would it be easier to read if there are markdown highlights for the user inputs and class names? 
e.g. `add homework`, or `Time`

yeyutong811 commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,383 @@
+### Logic component
+
+
+![LogicComponent](diagrams/LogicClass.png)
```
The words in this diagram seems really small, wonder if there is a better way to re-organize the boxes in the diagram?
![image](https://user-images.githubusercontent.com/60011226/97668477-016d1800-1abd-11eb-826b-07d9c60a7a6a.png)

yeyutong811 commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,383 @@
 # Developer Guide
 
+- [Design & implementation](#design--implementation)
```
Maybe you can consider to add numberings to the sections so that audience can follow better?

yeyutong811 reviewed (commented):
Overall seems quite neat!

QX-CHEN committed:
Import timetable

QX-CHEN committed:
Minor change

arindamshivatrip committed:
Implemented the import of ics files.

QX-CHEN committed:
Remove header color
Fix internal links

QX-CHEN committed:
Fix code style

QX-CHEN committed:
Fix date format example

mhchan163 committed:
Fixed reminder java.NullpointerException error

mhchan163 committed:
Added reminder to the add function

mhchan163 committed:
Added reminder to edit function

mhchan163 committed:
Added time conponent and timercanceler

mhchan163 committed:
Put timeCanceler to add timer when the timer object is created
moved isReminder to reminder class
Moved offReminder to task class

mhchan163 committed:
Added restartReminders method is storage
Changed TimerCanceler class to Timers

mhchan163 committed:
Added Reminder column to list format
Removed isReminder boolean from Reminder class
StartReminder throws InvalidReminderException when no start time or time is given

mhchan163 committed:
Refactor isOn boolean to Reminder class.
Set Reminder to instantiate whenever Task object is created,
now Timer object is only created when startReminder method is called.

mhchan163 committed:
Fix coding quality violations

mhchan163 committed:
fixed coding violations

mhchan163 committed:
...

mhchan163 committed:
Adjust JUnit tests accordingly

mhchan163 committed:
Added JUnit test in TaskTest for reminder function

QX-CHEN committed:
Merge pull request #76 from QX-CHEN/master

Import timetable

QX-CHEN committed:
Merge branch 'master' into master

mhchan163 committed:
Moved start and off Reminder methods to Reminder class

mhchan163 committed:
Merge branch 'master' of https://github.com/AY2021S1-CS2113T-W12-1/tp.git

# Conflicts:
#	src/main/java/seedu/messages/Messages.java
#	src/main/java/seedu/storage/Storage.java

mhchan163 committed:
Merge branch 'master' of https://github.com/mhchan163/tp.git

# Conflicts:
#	src/main/java/seedu/storage/Storage.java

QX-CHEN committed:
Merge pull request #110 from mhchan163/master

Integrated reminder into the add function

timcheahyi committed:
ug update, small

czlin98 committed:
Merge pull request #4 from AY2021S1-CS2113T-W12-1/master

merge back

mhchan163 committed:
Add reminder to add feature in Command Summary

mhchan163 committed:
Fixed help message

timcheahyi committed:
disallowed st > et

timcheahyi committed:
typos

QX-CHEN committed:
Fix file reading

czlin98 committed:
Allow spaces around command word and command argument.

czlin98 committed:
Allow spaces around command word and command argument.

QX-CHEN committed:
Move string conversion methods to Util

czlin98 committed:
Fix failed null args test cases.

mhchan163 committed:
Standardised the UG pictures and add reminder function to the
add command.

mhchan163 committed:
Standardised the UG pictures and add reminder function to the
add command.

mhchan163 committed:
Merge branch 'master' of https://github.com/mhchan163/tp.git

mhchan163 committed:
Made some changes to images

mhchan163 committed:
added more images

mhchan163 committed:
Fixed some errors in the images

mhchan163 committed:
Edited some text errors and fixed a few erroneous images

QX-CHEN committed:
Add JUnit test for Util

arindamshivatrip committed:
made changes to the ical reader to fix coding style errors and also made bug fixes that i assignned to myself.

arindamshivatrip committed:
added image

arindamshivatrip committed:
Cleaned up remaining coding standard exceptions in other files

QX-CHEN committed:
Minor change

QX-CHEN committed:
Merge pull request #112 from e0406981/master

UG changes and disallowed st>et

QX-CHEN committed:
Merge branch 'master' of https://github.com/AY2021S1-CS2113T-W12-1/tp

QX-CHEN committed:
Merge branch 'master' into master

QX-CHEN committed:
Merge pull request #113 from QX-CHEN/master

Fix file reading

QX-CHEN committed:
Merge pull request #115 from mhchan163/master

Updated UG with information of reminder and standardized images in UG

QX-CHEN committed:
Merge pull request #114 from czlin98/master

Allow spaces around command word and command argument.

QX-CHEN committed:
Merge branch 'master' into nusmod

QX-CHEN committed:
Merge branch 'nusmod' into master

QX-CHEN committed:
Merge pull request #116 from arindamshivatrip/master

updated code to fix the coding standard exceptions and made changes according to bug fixes i assigned to myself.

QX-CHEN committed:
Fix import timetable with reminder

QX-CHEN committed:
Merge branch 'nusmod'

QX-CHEN committed:
Fix list command parsing

QX-CHEN committed:
Merge pull request #117 from QX-CHEN/master

Fix list command parsing

QX-CHEN committed:
Fix issues in DG

QX-CHEN committed:
Fix code style

QX-CHEN committed:
Merge pull request #118 from QX-CHEN/master

Fix issues in DG

mhchan163 committed:
Added comments for reminder class and reminder related methods that require explanation

arindamshivatrip committed:
Merge pull request #1 from AY2021S1-CS2113T-W12-1/master

Merging

timcheahyi committed:
added JavaDoc for most commands

timcheahyi committed:
summary javadoc for edit execute

QX-CHEN committed:
Merge branch 'master' of https://github.com/mhchan163/tp into mhchan163-master

# Conflicts:
#	src/main/java/seedu/commands/DisplayReminderCommand.java

QX-CHEN committed:
Merge branch 'mhchan163-master'

QX-CHEN committed:
Merge pull request #120 from e0406981/master

added JavaDoc for most commands

timcheahyi committed:
updated return comments

timcheahyi committed:
updated return comments

QX-CHEN committed:
Add java docs for Utility class

QX-CHEN committed:
Merge branch 'master' into javaDocs

QX-CHEN committed:
Add java docs for TaskMap

QX-CHEN committed:
Add java docs for Model and DataSatck

QX-CHEN committed:
Update help command display and UG

czlin98 committed:
Merge pull request #5 from AY2021S1-CS2113T-W12-1/master

merge back

czlin98 committed:
New exception type for command format errors.

czlin98 committed:
Filled in empty javadoc clauses in add and edit commands.

czlin98 committed:
Added new invalid format exception to tests.

czlin98 committed:
Added new invalid format exception to tests again.

czlin98 committed:
Change exceptions in add tests from invalidcommand to invalidformat.

QX-CHEN committed:
Minor change in UGDG

QX-CHEN committed:
Merge pull request #122 from czlin98/format-exception

New format exception

QX-CHEN committed:
Merge branch 'master' of https://github.com/AY2021S1-CS2113T-W12-1/tp into javaDocs

QX-CHEN committed:
Fix code style

QX-CHEN committed:
Merge pull request #121 from QX-CHEN/javaDocs

Add java docs for Utility class

mhchan163 committed:
Edited DG use and manual testing, edited TaskMapClass uml diagram

mhchan163 committed:
Format editing

timcheahyi committed:
more javadoc

timcheahyi committed:
javadoc

timcheahyi committed:
javadoc

timcheahyi committed:
formatting

timcheahyi committed:
javadoc

timcheahyi committed:
javadoc

timcheahyi committed:
formatting

timcheahyi committed:
formatting

timcheahyi committed:
PPP

arindamshivatrip committed:
Merge pull request #2 from AY2021S1-CS2113T-W12-1/master

updating

QX-CHEN committed:
Merge pull request #124 from e0406981/master

mostly javadoc

czlin98 committed:
Add redo function.

czlin98 committed:
Add undo and redo in user guide.

czlin98 committed:
Merge branch 'master' into redo-function

czlin98 committed:
Fix explicit imports in parser.

czlin98 committed:
Fix undo.png

czlin98 committed:
Merge pull request #125 from czlin98/redo-function

Redo function

mhchan163 committed:
Added PPP

czlin98 committed:
Update help message and image.

czlin98 committed:
Remove reminder from user guide

czlin98 committed:
Merge pull request #127 from czlin98/redo-function

Update help message and image.

mhchan163 committed:
Added author tag for DG

mhchan163 committed:
Merge pull request #126 from mhchan163/master

Added PPP and updated DG

QX-CHEN committed:
Add ppp

QX-CHEN committed:
Merge pull request #128 from QX-CHEN/master

Add ppp

mhchan163 committed:
Minor changes

mhchan163 committed:
Merge branch 'master' of https://github.com/AY2021S1-CS2113T-W12-1/tp.git

QX-CHEN committed:
Update ppp link

QX-CHEN committed:
Update ppp page

QX-CHEN committed:
Merge pull request #129 from QX-CHEN/master

Update ppp

czlin98 committed:
Merge pull request #6 from AY2021S1-CS2113T-W12-1/master

merge back

QX-CHEN committed:
Fix github page issue

QX-CHEN committed:
Merge pull request #130 from QX-CHEN/master

Fix github page issue

QX-CHEN committed:
Update PPP

QX-CHEN committed:
Merge branch 'master' of https://github.com/AY2021S1-CS2113T-W12-1/tp

timcheahyi committed:
Update and rename TimothyCheah.md to e0406981.md

QX-CHEN committed:
Update aboutUs

czlin98 committed:
Merge pull request #7 from AY2021S1-CS2113T-W12-1/master

merge back

QX-CHEN committed:
Update ppp

mhchan163 committed:
Changed PPP slighty and update Glossary for DG

QX-CHEN committed:
Merge branch 'master' into master

QX-CHEN committed:
Merge pull request #131 from QX-CHEN/master

Update ppp

QX-CHEN committed:
Update aboutUs link

QX-CHEN committed:
Merge branch 'master' of https://github.com/AY2021S1-CS2113T-W12-1/tp

mhchan163 committed:
Minor changes to DG

QX-CHEN committed:
Merge pull request #132 from QX-CHEN/master

Update aboutUs link

mhchan163 committed:
More minor changes

mhchan163 committed:
Changed format for delete under Manual testing

mhchan163 committed:
Merge pull request #133 from mhchan163/master

Updated DG glossary and adjustments to Manual testing section

QX-CHEN committed:
Set theme jekyll-theme-cayman

QX-CHEN committed:
Fix page build

QX-CHEN committed:
Merge pull request #134 from QX-CHEN/master

Fix page build

czlin98 committed:
Add PPP

czlin98 committed:
Merge pull request #135 from czlin98/master

Add PPP

czlin98 committed:
Fix PPP links in AboutUs.md

czlin98 committed:
Merge branch 'master' into master

arindamshivatrip committed:
Made changes to code structure and added PPP.

czlin98 committed:
Merge pull request #136 from czlin98/master

Fix PPP links in AboutUs.md

arindamshivatrip committed:
Fixed some exceptions

czlin98 committed:
Update latest release link in user guide

czlin98 committed:
undo.png

czlin98 committed:
Merge branch 'master' of https://github.com/czlin98/tp

czlin98 committed:
Fix latest release link in user guide

czlin98 committed:
Merge pull request #138 from czlin98/master

Fix latest release link in user guide

arindamshivatrip committed:
Made changes to the About us and to the PPP . Also added JavaDoc comments to the Storage.java file.

QX-CHEN committed:
Merge branch 'master' of https://github.com/arindamshivatrip/tp into arindamshivatrip-master

# Conflicts:
#	docs/AboutUs.md
#	src/main/java/seedu/parser/CalParser.java

QX-CHEN committed:
Merge branch 'arindamshivatrip-master'

czlin98 committed:
undo.png

czlin98 committed:
Delete undo.png

czlin98 committed:
Fix undo.png image

czlin98 committed:
Merge remote-tracking branch 'origin/master'

czlin98 committed:
Remove setting reminder in content list

czlin98 committed:
Merge pull request #139 from czlin98/master

Fix UG final

timcheahyi committed:
cs2101

QX-CHEN committed:
Merge pull request #140 from e0406981/master

cs2101

czlin98 committed:
Update DeveloperGuide.md

czlin98 committed:
Update UserGuide.md
